metadata-service/factories/src/main/java/com/linkedin/gms/factory/common/LocalCassandraSessionConfigFactory.java

```diff
@@ -13,37 +13,37 @@
 @Configuration
 public class LocalCassandraSessionConfigFactory {
 
-  @Value("${CASSANDRA_DATASOURCE_USERNAME:cassandra}")
-  private String cassandraDatasourceUsername;
+  @Value("${cassandra.datasourceUsername}")
+  private String datasourceUsername;
 
-  @Value("${CASSANDRA_DATASOURCE_PASSWORD:cassandra}")
-  private String cassandraDatasourcePassword;
+  @Value("${cassandra.datasourcePassword}")
+  private String datasourcePassword;
 
-  @Value("${CASSANDRA_HOSTS:cassandra}")
-  private String cassandraHosts;
+  @Value("${cassandra.hosts}")
+  private String hosts;
 
-  @Value("${CASSANDRA_PORT:9042}")
-  private String cassandraPort;
+  @Value("${cassandra.port}")
+  private String port;
 
-  @Value("${CASSANDRA_DATACENTER:datacenter1}")
-  private String cassandraDataCenter;
+  @Value("${cassandra.dataCenter}")
```

Comment:
I think this translates to CASSANDRA_DATA_CENTER in environment variables.
```suggestion
  @Value("${cassandra.datacenter}")
```